__tests__/app.test.js

```diff
@@ -24,10 +26,27 @@ describe("GET /api/topics", () => {
   });
 });
 
+
 describe("GET /api", () => {
   test("Returns an object describing all the available endpoints on your API", async () => {
     const response = await request(app).get("/api");
     expect(response.status).toBe(200);
     expect(response.body).toEqual(endpoints);
   });
 });
+
+
+describe("GET /api/articles/:article_id", () => {
+  test("should return the article object for a valid article_id", async () => {
+    const articleId = 1;
+    const article = await getArticleById(articleId);
```

Comment:
rather than test the function itself, we want to test the endpoint /api/articles/:article_id using supertest - similar to how you've test /api or /api/topics